docs/source/bentocloud/how-tos/manage-deployments.rst

```diff
@@ -40,41 +42,83 @@ To retrieve details about a specific Deployment:
       # To output the details in YAML (Default)
       bentoml deployment get <deployment-name> -o yaml
 
+    Expected output in YAML:
+
+    .. code-block:: yaml
+
+        name: summarization
+        bento: summarization:ghfvclwp2kwm5e56
+        cluster: aws-ca-1
+        endpoint_url: https://summarization-test--aws-ca-1.mt1.bentoml.ai
+        admin_console: https://test.cloud.bentoml.com/deployments/summarization/access?cluster=aws-ca-1&namespace=test--aws-ca-1
+        created_at: '2024-02-20 09:27:52'
+        created_by: bentoml-user
+        config:
+          envs: []
+          services:
+            Summarization:
+              instance_type: cpu.2
+              scaling:
+                min_replicas: 1
+                max_replicas: 2
+              envs: []
+              deployment_strategy: Recreate
+              extras: {}
+              config_overrides:
+                traffic:
+                  timeout: 10
+        status:
+          status: running
+          created_at: '2024-02-20 09:27:52'
+          updated_at: '2024-02-21 05:46:18'
+
   .. tab-item:: Python API
 
+    To get basic information of a Deployment:
+
     .. code-block:: python
 
-      bentoml.deployment.get(name="deployment-1")
-
-Expected output in YAML:
-
-.. code-block:: yaml
-
-    name: summarization
-    bento: summarization:ghfvclwp2kwm5e56
-    cluster: aws-ca-1
-    endpoint_url: https://summarization-test--aws-ca-1.mt1.bentoml.ai
-    admin_console: https://test.cloud.bentoml.com/deployments/summarization/access?cluster=aws-ca-1&namespace=test--aws-ca-1
-    created_at: '2024-02-20 09:27:52'
-    created_by: bentoml-user
-    config:
-      envs: []
-      services:
-        Summarization:
-          instance_type: cpu.2
-          scaling:
-            min_replicas: 1
-            max_replicas: 2
-          envs: []
-          deployment_strategy: Recreate
-          extras: {}
-          config_overrides:
-            traffic:
-              timeout: 10
-    status:
-      status: running
-      created_at: '2024-02-20 09:27:52'
-      updated_at: '2024-02-21 05:46:18'
+      import bentoml
+
+      dep = bentoml.deployment.get(name="deploy-1")
+      print(dep)
```

Comment:
I think we should introduce `dep.to_dict()` here, since this method will include everything like config and endpoint_urls in deployment.

Reply:
Added.